src/main/java/seedu/contax/model/appointment/Appointment.java

```diff
@@ -42,6 +43,29 @@ public Appointment(Name name, StartDateTime startDateTime, Duration duration, Pe
         this.startDateTime = startDateTime;
         this.duration = duration;
         this.person = person;
+        this.priority = null;
+    }
+
+    /**
+     * Additional Constructs an {@code Appointment}.
+     * The fields requirements follows the previous
+     * The fields {@code priority} must be present and not null.
```

Comment:
Previous does not refer to anything here.

Reply:
thx modified